Code/FooBar.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+class FooBar
+{
+    public function Checker($Number): string
```

Comment:
Still doesn't comply even PSR-1, look into method naming and argument naming guidelines.